src/tools/map_extractor/System.cpp

```diff
@@ -1025,6 +1026,48 @@ void ExtractDBCFiles(int locale, bool basicLocale)
     printf("Extracted %u DBC files\n\n", count);
 }
 
+void ExtractCameraFiles(int locale, bool basicLocale)
```

Comment:
This function should access the camera dbc and extract only models pointed by that dbc - reason for this is porting to 6.x, CASC does not have a listfile and you cannot iterate a directory to find cameras